administration/src/project-configs/koblenz/activationText.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, Typography, styled } from '@mui/material'
```

Comment:
Just copied from bavaria and may be adjusted. This will be opened if someone clicks on the qr code in the pdf and hasn't installed the app

Reply:
added TODO and created issue here
https://github.com/digitalfabrik/entitlementcard/issues/1676